Dockerfile.processing

```diff
@@ -8,21 +8,22 @@ RUN apt-get update && \
 	apt-get upgrade -y && \
 	apt install software-properties-common -y && \
 	add-apt-repository ppa:deadsnakes/ppa && \
-	apt-get install -y python3.10 python3-pip
+	apt-get update && apt-get install -y python3.10 python3.10-dev python3.10-venv && \
+    wget https://bootstrap.pypa.io/get-pip.py && \
+    python3.10 get-pip.py
 
 RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.10 1 \
```

Comment:
it sounds like this doesn't actually work. Id' try removing and see if everything still works.

Reply:
otherwise you should be able to use `python` instead of `python3.10` to start the interpreter